### DISCUSSION TITLE
Flow : read directus_user

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#14738] [Archive]
I added a new field in directus_user (shop id).
But in a flow, I can't read the collection directus_user.
Is is this something intented ? Is there a workaround ?

**Comment by nervusdm** [comment]
Ok I auto answer, maybe someone will one day fill this usefull. just click on the {} in the collection and manually write "directus_users".
(with full permission ofc)
<img width="960" alt="Capture d’écran 2022-07-29 à 10 27 31" src="https://user-images.githubusercontent.com/4217180/181717778-256b56a2-ef67-4afb-b88f-9104a5b21cb6.png">

**Reply by azrikahar** [reply]
Technically not all system tables can be used this way, so do note that some may not behave as expected. That is why they are not directly selectable at the moment (more info about this here: https://github.com/directus/directus/issues/14597#issuecomment-1193284731). However there is #13716 acting as the feature request for this use case 👍
